test-app/.editorconfig

```diff
@@ -0,0 +1,22 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.js]
+indent_size = 4
```

Comment:
Note that I just copied this file from the root of mlxprs, so if we change it here, I would want to change it there as well.
That's fine, but it would reformat the entire project.
Still want me to change it?

Reply:
Got it, that makes sense, so no change.

Reply:
Since it's like this in the root MLXPRS project, I'm pushing this issue for later.